microsoftgraph/docs-conceptual/use-query-parameters.md

```diff
@@ -27,7 +27,7 @@ Microsoft PowerShell SDK cmdlets may support one or more of the following OData
 | [-Filter](#filter-parameter)| Filters results (rows)|`Get-MgUser -Filter "startsWith(DisplayName, 'Conf')"`|
 | [-OrderBy](#orderby-parameter)| Orders results|`Get-MgUser -OrderBy DisplayName`|
 | [-Search](#search-parameter)| Returns results based on search criteria|`Get-MgUser -ConsistencyLevel eventual -Search '"DisplayName:Conf"'`|
-| [-Select](#select-parameter)| Filters properties (columns)|<code>Get-MgUser &#124; Select DisplayName, Id</code>|
+| [-Select](#select-parameter)| Filters properties (columns)|<code>Get-MgUser -Property Id, DisplayName &#124; Select Id, DisplayName</code>|
```

Comment:
```suggestion
| [-Property](#property-parameter)| Filters properties (columns)|<code>Get-MgUser -Property Id, DisplayName &#124; Select Id, DisplayName</code>|
```

Reply:
Can we also change the [`select` parameter section](https://github.com/MicrosoftDocs/microsoftgraph-docs-powershell/blob/main/microsoftgraph/docs-conceptual/use-query-parameters.md#select-parameter) to talk about and reference `-Property` parameter instead as this is the parameter's official name. `Select` is just an alias. For example:
1. The title should be changed from `Select parameter` to `Property parameter`.
2. Content should be updated to show `-Property`. For example, `Use the -Select query parameter to return a set of properties...` should be `Use the -Property query parameter to return a set of properties...`.